Automattic/wp-calypso
# Add backup features to checkout

#### Proposed Changes

Add a feature list of VaultPress backup when it is the only item in the cart

P2: p1HpG7-k8M-p2

#### Testing Instructions

1. Check out these changes via `git switch add/backup-features-to-checkout`
2. Start your local environment via `yarn start`
3. Go to https://cloud.jetpack.com/pricing and add VaultPress Backup (10GB) to your cart
4. When you are redirected to the cart, replace `https://wordpress.com` with `http://calypso.localhost:3000/` in the URL
5. The new feature list should now show up above the total
![image](https://user-images.githubusercontent.com/65001528/214434996-c4e0f293-c95b-45d9-8494-4f27e5b78e4f.png)
6. If you'd like to run the new tests manually, running ` yarn test-client client/my-sites/checkout/composite-checkout/test/wp-checkout-order-summary.tsx` from the root directory should do the trick

#### Pre-merge Checklist
